src/r128_driver.c

```diff
@@ -94,7 +94,11 @@
 #include "xf86RandR12.h"
 #include "xf86cmap.h"
 #include "xf86xv.h"
+#if defined(__i386__) || defined(__x86_64__)
```

Comment:
It would be nice to define a single macro and then use it instead of repeating this or. Not a big deal though.

Also, I’m not sure, do we support any other compilers than gcc, do we need to support other arch macros: https://sourceforge.net/p/predef/wiki/Architectures/

Reply:
I think those macros should be available with clang, so we’re fine. MSVC probably defines something different, but I guess we don’t need it?